docs/UserGuide.md

```diff
@@ -182,154 +180,309 @@ Example:
 
-Format: `editsetup [n/USERNAME] [sd/STARTDURATION] [ed/ENDDURATION] [fy/FISCALYEAR]`
+Format: `editsetup [un/USERNAME] [sd/STARTDURATION] [ed/ENDDURATION] [ay/ACADEMICYEAR]`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-Edit the duration of the semester with either start duration or end duration or both
+Items in square brackets are optional, but please input at least 1 value to change.
 </div>
 
 Examples:
-*  `editsetup n/Jane Doe sd/23-02-2020 ed/29-06-2020 fy/AY19/20 Sem 2`
-*  `editsetup n/Jenny Doe sd/23-02-2020 fy/AY19/20 Sem 2`
+*  `editsetup un/Jane Doe sd/23-02-2020 ed/29-06-2020 ay/AY19/20 Sem 2`
+*  `editsetup un/Jenny Doe ay/AY19/20 Sem 2`
 
 ### Delete Setup Details: `deletesetup`
 
 Delete all setup details in GrAB3.
 
 Format: `deletesetup` 
 
-### Add a Task to Task Schedule: `addtask`
+### Add a User Task to Task Schedule: `addusertask`
 
-Add a task to the task schedule in GrAB3.
+Add a user task to the task schedule in GrAB3.
 
-Format: `addtask n/NAME sd/DATE d/DURATION [t/TAG]`
+Format: `addusertask tn/TASKNAME du/DURATION da/DAY w/WEEK [t/TAG]`
 
-- Addition of a task to a specific day will be **rejected** if the total tasks for that day exceed 24 hours.
+- `DURATION` will be in hours and less than 24 hours
+- `DAY` have to be an integer within 1-7, with 1 being monday and 7 being sunday
+- `WEEEK` have to be a valid positive integer. Total number of week calculate from semester start and end date. <br/>
+    e.g. If the semester is from 01 to 25/09/2020 <br/>
+            * Total week in the semester : 4 <br/>
+            * Week 1 will be from 01/09/2020 to 06/09/2020 <br/>
+    `addusertask tn/CCA lunch meeting dur/4 day/2 w/1` will add user task to Tuesday of Week 1 (01/09/2020)
+- Addition of a user task to a specific day will be **rejected** if the total tasks for that day exceed 24 hours
 
 Examples:
-* `addtask n/John Doe sd/20-09-2020 d/2`
-* `addtask n/John Doe sd/20-09-2020 d/2 t/CS2101`
+* `addusertask tn/CCA lunch meeting du/4 da/2 w/1`
+* `addusertask tn/CCA lunch meeting du/4 da/2 w/2 t/CCA`
 
-### Edit Task Details: `edittask`
+### Edit User Task Details: `editusertask`
 
-Edit task details for GrAB3, including task name, date, duration and tag.
+Edit user task details for GrAB3, including task name, duration, day, week and tag.
 
-Format: `edittask i/INDEX n/TASKNAME sd/DATE d/DURATION`
+Format: `editusertask id/TASKID [tn/TASKNAME] [du/DURATION] [da/DAY] [w/WEEK] [t/TAG]`
 
-- The index must be a **positive integer** and must be **valid**
-- `DURATION` must be in hours
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Items in square brackets are optional, but please input at least 1 value to change.
+</div>
+
+- `DURATION` will be in hours and less than 24 hours
+- `DAY` have to be an integer within 1-7, with 1 being monday and 7 being sunday
+- `WEEEK` have to be a valid positive integer. Total number of week calculate from semester start and end date. <br/>
```

Comment:
*has
Can omit the second line if we are providing them with the start and end dates?